client/src/utils/functions.js

```diff
@@ -14,3 +14,38 @@ export function toIso(europeanDate) {
   const [day, month, year] = europeanDate.split("/");
   return `${year}-${month}-${day}`;
 }
+
+export function budgetFormValidator(formData) {
+  const errors = {};
+  const specialCharacters = /[^A-Za-zÀ-ÿ0-9 ]/;
```

Comment:
**praise**: That's way better ! Way easier to understand (even for me) and no more unecessary option such as `gu` 👏 

**question**: So let's try one more time, without looking at any resources or testing your code:
- What is that string ?
- What does it do ?
- What are those `/` purpose ?

Reply:
That string is a regex. A regular expression. It is supposed to target specific characters. Here, everything after `^` and before `]` is what will be allowed in my inputs. Because ^ means "all that is not `A-Za-zÀ-ÿ0-9` is selected".
To "escape" code (not sure of the english word), so that it is not considered javascript syntax between the two `/`

Reply:
Okay, knowing that, do you think a variable named `specialCharacters` fits the description that you gave me